### PR TITLE
Amend identify needs (Additional information)

### PR DESCRIPTION
**Pull request by JakeGoodwin22**
## What does this pull request do?

Allow PP to modify the additional information (Identify needs)

## What is the intent behind these changes?

Allows a PP to modify details of a referral which has been submitted, preventing them having to close and make a new referral. This change is for modifying the additional information

### CODE REVIEW

**Comment by kth13** [comment]
cypress/downloads/downloads.html is added, do we really need this?

File: server/routes/amendAReferral/additionalInformation/amendAdditionalInformationForm.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  static readonly reasonForChangeId = 'reason-for-change'
+
+  constructor(private readonly request: Request) {}
+
+  static get validations(): ValidationChain[] {
+    return [
+      body(AmendAdditionalInformationForm.reasonForChangeId)
+        .notEmpty({ ignore_whitespace: true })
+        .withMessage(errorMessages.amendReferralFields.missingReason),
+      body(AmendAdditionalInformationForm.additionalInformationId)
```
Comment (kth13) [inline server/routes/amendAReferral/additionalInformation/amendAdditionalInformationForm.ts:21]: additionalInformation can be empty as it is an optional field

**Comment by JakeGoodwin22** [comment]
> cypress/downloads/downloads.html is added, do we really need this?

No idea where that came from, removed thanks!

**Review by kth13 — APPROVED** [review]
LGTM

**Pull request merged**